src/lib/templateengine.js

```diff
@@ -1394,6 +1391,14 @@ function TemplateEngine( undefined ) {
         });
       }
     }
+    return page_id;
+  }
+
+  this.scrollToPage = function(page_id, speed, skipHistory) {
+    if( undefined === page_id )
+      page_id = this.screensave_page;
+    page_id = thisTemplateEngine.getPageIdByName(page_id);
```

Comment:
When the parameter of scrollToPage is not the page_id I would rename it to avoid confusion